torch/backends/mps/__init__.py

```diff
@@ -15,12 +15,16 @@ def is_built() -> bool:
 @_lru_cache()
 def is_available() -> bool:
     r"""Returns a bool indicating if MPS is currently available."""
+    if not hasattr(torch._C, '_is_mps_available'):
+        return False
     return torch._C._is_mps_available()
 
 
 @_lru_cache()
 def is_macos13_or_newer() -> bool:
     r"""Returns a bool indicating whether MPS is running on MacOS 13 or newer."""
+    if not hasattr(torch._C, '_is_mps_on_macos_13_or_newer'):
+        return False
```

Comment:
Again, shouldn't be needed, though I wonder if this call should really trigger an exception on non-MPS capable systems:
```
$ python -c "import torch;import sys;print(sys.platform, torch.__version__, torch.backends.mps.is_macos13_or_newer())"
linux 2.0.0a0+git208f330 False
```
```suggestion
```

Reply:
Updated the PR.